include/deal.II/base/process_grid.h

```diff
@@ -217,6 +233,20 @@ namespace Utilities
     }
 
 
+    inline int
```

Comment:
Here and below: We usually have three blank lines instead of two between function definitions.